docs/en/ingest-management/tab-widgets/install-layout.asciidoc

```diff
@@ -14,6 +14,8 @@ Log files for {beats} shippersfootnote:lognumbering[]
 `/usr/bin/elastic-agent`::
 Shell wrapper installed into PATH
 
+You can install {agent} in a custom base path that's different from `/Library`.  When installing {agent} with the `./elastic-agent install`
+command, provide the custom base path via the `--base-path` CLI option.
```

Comment:
```suggestion
You can install {agent} in a custom base path other than `/Library`.  When installing {agent} with the `./elastic-agent install`
command, use the `--base-path` CLI option to specify the custom base path.
```
Just a small change to avoid `via` (we try not to use Latin expressions to help non-native English speakers and translatability).